### DISCUSSION TITLE
Can I use lossless to without encoding merge audio back into video after using audacity?

### DISCUSSION THREAD

**Discussion by giddyhara** [mifi/lossless-cut#1760] [General]
How can I combine the 2 again as if there were the original with lossless? (so same length, no desync)? Can I do this with lossless or is there an other tool to combine without encoding?

**Comment by mifi** [comment]
That should be possible yes. You do it in separate steps

**Reply by giddyhara** [reply]
Could you explain the steps? Because you say "should" as in you dont know for sure. I first add the video, then an an audio track. But I dont see any way to fix a possible desync if that would happen. I just dont want to manually align video and audio... I dont want to edit aaudio in audacity only to waste my time getting a desync which I cant fix.

**Comment by mifi** [comment]
should as in: it works with many formats/codecs, but i cannot guarantee that it works with your particular file and you have to test it to see if the result is satisfactory. there is no way to fix any desync if it should happen. if you don't want to waste time editing in audacity and risk it not working in llc, then you should not do it.
